sbin/common/config_init.sh

```diff
@@ -81,6 +81,8 @@ OPENJDK_BUILD_NUMBER
 OPENJDK_CORE_VERSION
 OPENJDK_FEATURE_NUMBER
 OPENJDK_FOREST_NAME
+OPENJDK_FOREST_SOURCE_ARCHIVE
```

Comment:
Archive to me usually means .zip or .tar etc.  Is OPENJDK_FOREST_SOURCE or something OPENJDK_LOCAL_SOURCE a better name?

Reply:
It can be archive archive.   You can build local directory and tarball  (both with sources) in same way.
As i replied to @sxa https://github.com/adoptium/temurin-build/pull/3737#issuecomment-2034530869 the  proepr name aka `OPENJDK_LOCAL_SOURCE_DIR_OR_ARCHIVE` is terrible, but why not.
```
sh makejdk-any-platform.sh  -o ~/git/java-21-openjdk-portable/openjdk-21.0.2+13.tar.xz  jdk21u
```
and
```
 sh makejdk-any-platform.sh -o  ~/git/jdk21u/ jdk21u
```

both do same job.

Reply:
actually, `OPENJDK_LOCAL_SOURCE` is generic enough. @sxa  will use that.